cms/templates/admin/base_site.html

```diff
@@ -15,5 +15,5 @@ <h1 id="site-name"><a href="{% url 'admin:index' %}">{{ site_header|default:_('D
             <a href="{{ docsroot }}">{% trans 'Documentation' as tmsg %} {{tmsg|force_escape}}</a> /
         {% endif %}
     {% endif %}
-    <a href="{% url 'admin:logout' %}">{% trans 'Log out' as tmsg %} {{tmsg|force_escape}}</a>
+    <a href="/logout/">{% trans 'Log out' as tmsg %} {{tmsg|force_escape}}</a>
```

Comment:
Hmm, I guess you can't use `url 'logout'` on this one, since that produces the version without the trailing slash -- and we need the one that does have the trailing slash (which is from auth-backends). Ugh.